core/XH.ts

```diff
@@ -326,6 +330,16 @@ export class XHApi {
         return this.identityService?.username ?? null;
     }
 
+    /**
+     * Logout the current user.
+     * @see HoistAuthModel.logoutAsync
+     */
+    async logoutAsync(): Promise<void> {
+        await this.authModel?.logoutAsync();
+        await this.appModel?.logoutAsync();
```

Comment:
Do we still want to support AppModel.logoutAsync(), given that apps can now provide their own HoistAuthModel implementation that handles any custom logic? Should that be the preferred method going forward?

Reply:
Yes, we can go ahead and remove that for clarity.